doc/pictures/testclient_basic.cast

```diff
@@ -1,22 +1,22 @@
 {"version": 2, "width": 80, "height": 33, "ts": 1599119755, "env": {"SHELL": "/usr/bin/fish", "TERM": "rxvt-unicode-256color"}}
 [0.028694, "o", "\u001b]0;fish /home/wenwen/tools/kuksa.val"]
```

Comment:
If we change these can we at least also re-render them? See https://github.com/eclipse/kuksa.val/blob/master/doc/pictures/README.md

(shall we also redo them (e.g. record fresh .casts) maybe, but if not super easy maybe not in this PR)

Reply:
I re-rendered the GIFs.
Let's keep re-recording for later I would say.